main.nf

```diff
@@ -1014,8 +1090,32 @@ process createGEM {
  */
 
 
+
```

Comment:
So this section confuses me.  Why is this showing up as newly added code in this PR when it was added previously.  This is one example of why I get nervous with Git sometimes....  I can see above where you have made changes that make sense for Aspera integration but this shouldn't be here. It should already be part of master....

Reply:
If you're referring to the fastq_dump process, git shows it as being replaced rather than added. It's just that the deletions of the old code are scattered throughout the new additions, but if you look through you'll see that all of the pieces of the original code are there.

Reply:
I couldn't get all of the code to show up in the box above... But its lines 1094-1116.  It includes the clean_sra process.  It's showing green as if you added it new, but it's not new.  This is what makes me nervous bout git sometimes.  Why did it do this, where is getting this code from and will it make changes.

Reply:
No `clean_sra` was just added in this branch. You may be confusing it with one of the other cleanup processes. The `fastq-dump` tool caches SRA files in a separate directory (`$USER/ncbi/public` by default) so they were never seen in the work directory. Now that we use `ascp` or `prefetch` to download SRA files before extracting them, we have to put them in the work directory and so now they're visible.

Reply:
Oh, you're right.  Sorry I read clean_sra and my brain thought clean_sam.   It's good.